hummingbot/strategy/avellaneda_market_making/avellaneda_market_making.pyx

```diff
@@ -614,14 +614,15 @@ cdef class AvellanedaMarketMakingStrategy(StrategyBase):
 
             self.c_collect_market_variables(timestamp)
             if self.c_is_algorithm_ready():
-                proposal = None
                 if self._create_timestamp <= self._current_timestamp:
                     # Measure order book liquidity
                     self.c_measure_order_book_liquidity()
 
                 self._hanging_orders_tracker.process_tick()
+
+                # Needs to be executed at all times to not to have active order leftovers after a trading session ends
                 self.c_cancel_active_orders_on_max_age_limit()
-                self.c_cancel_active_orders(proposal)
+
```

Comment:
Iirc, Isnt the `c_is_within_tolerance` function called within `c_cancel_active_orders()`? Removing it will mean that the `order_refresh_tolerance_pct` parameter will not be taken into consideration.

Reply:
the c_cancel_active_orders() was there twice, in this case the proposal was always None which caused the orders to be cancelled all the time regardless of the tolerance